packages/foundation-models/package.json

```diff
@@ -20,24 +20,27 @@
     "internal.d.ts"
   ],
   "scripts": {
-    "compile": "tsc",
+    "compile": "pnpm run generate-zod && tsc",
```

Comment:
[q] is it really needed for transpilation?

Reply:
I think it just ensures a bit more that if someone changes `azure-openai-types.ts`, the schema is generated again if they compile it. Not super necessary but won't hurt at least.